README.md

```diff
@@ -359,6 +359,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Ankit Singh](https://github.com/ankitsingh241)
   - [Ankit Warbhe](https://github.com/ankitwarbhe)
   - [Ankit Zore](https://github.com/ZoreAnkit)
+  - [AlexxTheCurator](https://github.com/AlexxTheCurator)
```

Comment:
Please relocate your name and insert it on line 300. Name should be in alphabetical order.

Reply:
Done